### DISCUSSION TITLE
Improvement ideas

### DISCUSSION THREAD

**Discussion by Andi1986** [marcusolsson/grafana-gantt-panel#7] [Ideas]
Hi,
i have tested the plugin and it works good so far. Credit to the author !
My goal is to have a Cron visualization which shows, when was a cron running and for how long was it running. This is very well visualisable with Gantt. 
Now i have a few things in mind that should be implmented in my opinion to make this work.
- There should be another dimension value which declares the color for each activity (green, yellow, red) so we can for example see that one Cron is still running, or a Cron has actually failed with an exception
- and also there should be another dimension where we can declare our own hover text, when hovering over the activity (for example to show the error message when a cron failed)
- in other graphs of grafana it is possible to zoom in, insiade the graph for having a shorter time range by just marking a certain part of the graph. This should also be possible for the Gantt graph.

in the current hovertext we see start and end time but this is shown as the raw timestamp value coming from source, is it possible to format it for readable time ?

Maybe have more ideas to come but these are most important for me right now. Tell me what you think about

thanks and greets
Andi

p.s. im using ElasticSearch data source

**Comment by marcusolsson** [comment]
Thank you for trying out the plugin! I'm glad it's working well for you.

I think all your ideas are well within what you'd expect from this plugin. In particular, I've been wanting to add your first two ideas since day 1, but I've lacked a good use case, which you've kindly provided!

Would you mind creating three separate issues for your ideas so that I can add them to the [project board](https://github.com/users/marcusolsson/projects/1), and so we can discuss them individually?

Thanks,
Marcus
